raiden/tests/integration/rpc/test_assumptions_parity.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+from raiden.tests.integration.rpc.test_assumptions import deploy_rpc_test_contract
+from raiden.utils import safe_gas_limit
+
+pytestmark = pytest.mark.usefixtures('skip_if_not_parity')
+
+# set very low values to force the client to prune old state
+STATE_PRUNNING = {
```

Comment:
so far from the chat we had in rocketchat seems that you have not found a way to replicate pruned state in Geth right?